DigitalLearningSolutions.Web/Views/TrackingSystem/Centre/Shared/_CentreSideNavMenu.cshtml

```diff
@@ -0,0 +1,36 @@
+﻿@using DigitalLearningSolutions.Web.Helpers
+@using DigitalLearningSolutions.Web.Helpers.TrackingSystem.Centre
+@using Microsoft.Extensions.Configuration
+@inject IConfiguration Configuration
+@model CentrePage
+
+@{
+  const string selectedClass = "side-nav__item--selected";
+
+  var dashboardItemClass = Model == CentrePage.Dashboard ? selectedClass : string.Empty;
+  var configurationItemClass = Model == CentrePage.Configuration ? selectedClass : string.Empty;
+  var administratorsItemClass = Model == CentrePage.Administrators ? selectedClass : string.Empty;
+  var reportsItemClass = Model == CentrePage.Reports ? selectedClass : string.Empty;
+}
+
+
+<h2 class="side-nav__heading">Centre</h2>
+<ul class="nhsuk-list side-nav__list">
+  <li class="side-nav__item @dashboardItemClass" aria-current="@(Model == CentrePage.Dashboard ? "page" : null)">
+    <a class="side-nav__link" asp-action="Index" asp-controller="Dashboard">Centre dashboard</a>
+  </li>

```

Comment:
Do you think it's worth creating a VC for this like we've done with the text inputs to avoid the XXXItemClass and aria-current logic duplication, but still keep text in HTML?

Reply:
Perhaps. It's a toss up though, as I think having them as a VC would mean losing some nice validation functionality on the asp-action and asp-controller attributes. Probably worth it as that can probably be easily tested anyway.

Reply:
I'll leave it up to you 👍